### PR TITLE
Fix workbench issue in backported 1.11 branch: error message cannot display

### PR DESCRIPTION
**Pull request by chloe-zh**
*Issue #, if available:*

*Description of changes:*

- Fixed error message display issue
- Updated unit test

**Before:** 
image

**After:** 
image

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench/public/components/Main/main.tsx [hunk 1]
```diff
@@ -260,13 +261,17 @@ export class Main extends React.Component<MainProps, MainState> {
...
       return {
         fulfilled: false,
-        errorMessage: response.data.resp,
-        data: response.data.body,
+        errorMessage: error.data.resp,
+        data: error.data.body,
       };
     }
 
+    // response: IHttpResponse<ResponseData>
```
Comment (davidcui-amzn) [inline workbench/public/components/Main/main.tsx:274]: Do we need to leave these comments in?

**Review by davidcui-amzn — APPROVED** [review]
LGTM- one minor comment

**Review by penghuo — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

File: workbench/public/components/Main/main.tsx (hunk 1 above)
Comment (chloe-zh) [reply workbench/public/components/Main/main.tsx:274]: Yes I added this comment here to remind us that the responses of success and error are of different types

File: workbench/public/components/Main/main.tsx [hunk 2]
```diff
@@ -251,7 +251,8 @@ export class Main extends React.Component<MainProps, MainState> {
     };
   }
 
-  processQueryResponse(response: IHttpResponse<ResponseData>): ResponseDetail<string> {
+  processQueryResponse(response: any): ResponseDetail<string> {
+    console.log(response)
```
Comment (joshuali925) [inline workbench/public/components/Main/main.tsx:255]: unnecessary console.log?

Comment (chloe-zh) [reply workbench/public/components/Main/main.tsx]: removed, thanks!

**Pull request merged**